src/components/review/ProposalTechnicalReview.tsx

```diff
@@ -226,7 +226,7 @@ const ProposalTechnicalReview = ({
                 type="submit"
                 onClick={() => setShouldSubmit(false)}
                 variant="contained"
-                color="secondary"
+                color={isUserOfficer ? 'primary' : 'secondary'}
```

Comment:
For Instrument scientist we have two buttons `Update` and `Submit` and for user officer we have only `Update` so I thought it will be better to have secondary color on update button if we have two buttons and primary if it is only one button.